src/jumio-api/interfaces/jumio-transaction-retrieve.ts

```diff
@@ -102,9 +113,31 @@ export type ImageCheck = {
   data: {
     faceSearchFindings: {
       status: string;
```

Comment:
I discovered this is an optional return field in their API, it isn't present if there are no findings, but is present and is a list if there is.

Reply:
(this is undocumented in their API)